daemon/k8s_watcher.go

```diff
@@ -275,7 +275,7 @@ func (d *Daemon) addK8sNetworkPolicy(obj interface{}) {
 	}
 	rules, err := k8s.ParseNetworkPolicy(k8sNP)
 	if err != nil {
-		log.WithError(err).WithField(logfields.Object, logfields.Repr(obj)).Error("Error while parsing kubernetes network policy")
+		log.WithError(err).WithField(logfields.Object, logfields.Repr(obj)).Error("Error while parsing k8s network policy")
```

Comment:
The `obj` is an `interface{}`, isn't the output different if it's not the `k8sNP`?

Reply:
Good point. I was surprised to find that it wouldn't matter (https://play.golang.org/p/VAcbAhMobS) but I redid it so each message refers to the variable name it was trying to work with.